config/DatabaseSetup.sql

```diff
@@ -25,27 +45,58 @@ FOREIGN KEY (ProductID) REFERENCES Products(ProductID)
 ON DELETE CASCADE
 );
 
-ALTER TABLE Runs AUTO_INCREMENT=501;
+ALTER TABLE Runs AUTO_INCREMENT=301;
 
 
 CREATE TABLE Batches (
 RunID int,
-Amount float(10),
+Amount int,
 Location varchar(100),
 FOREIGN KEY (RunID) REFERENCES Runs(RunID)
 ON DELETE CASCADE
 );
 
-ALTER TABLE BATCHES AUTO_INCREMENT=701;
+CREATE TABLE Logs (
+LogID int AUTO_INCREMENT,
+LogType int,
+ProductID int,
+UserID int,
+CustomerID int,
+Time datetime,
+GenericVar int,
+PRIMARY KEY (LogID),
+FOREIGN KEY (ProductID) REFERENCES Products(ProductID),
+FOREIGN KEY (UserID) REFERENCES Users(UserID),
+FOREIGN KEY (CustomerID) REFERENCES Customers(CustomerID)
+);
+
+ALTER TABLE Logs AUTO_INCREMENT = 1001;
+
 
 ##### Fill with sample data #####
 
+## Users(UserID, Name, Email, PermissionLevel, Password) ##
+INSERT INTO Users
+VALUES (NULL, "Rick", "rick@gmail.com", null, "rickster"),
+	   (NULL, "Kelly", "kgirl22@yahoo.com", null, "password"),
+       (NULL, "Donovan", "dman@hotmail.com", null, "donovan"),
+       (NULL, "Han Solo", "igosolo@pdx.edu", null, "fuzzball");
+
+## Customers(CustomerID, Name) ##
+INSERT INTO Customers
+VALUES (NULL, "Mercy Corps"),
+	   (NULL, "Great Business"),
+       (NULL, "Sweet Shoes"),
+       (NULL, "Poster Professionals");
+
+## Proudcts(ProductID, Name, Customer, Description, DateCreated) ##
 INSERT INTO Products
-VALUES (NULL, "Business Cards", "Mercy Corps", "Super slick business cards", "2015-05-03"),
-	   (NULL, "PSU Stationary", "Portland State", "Letterhead and stationary for PSU", "2015-05-07"),
-       (NULL, "Envelopes", "Warren", "Warren's customized capstone envelopes, 5 x 8", "2015-05-18"),
-       (NULL, "IMP Display Board", "TeamC", "Project display board, 6 x 10", "2015-05-18");
+VALUES (NULL, "Business Cards", 401, "Super slick business cards", "2015-05-03"),
```

Comment:
We will want a todo (or a JIRA ticket if a todo isnt possible) to remind us to take out this prepopulated data at a later point.